src/Interfacer.jl

```diff
@@ -85,10 +89,19 @@ function Base.show(io::IO, sim::CoupledSimulation)
         "Coupled Simulation\n",
         "├── Running on: $(device_type)\n",
         "├── Output folder: $(sim.dirs.output)\n",
-        "└── Current date: $(sim.dates.date[])",
+        "└── Current date: $(current_date(sim))\n",
     )
 end
 
+"""
+    current_date(cs::Interfacer.CoupledSimulation)
+
+Return the model date at the current timestep.
+# Arguments
+- `cs`: [CoupledSimulation] containing info about the simulation
+"""
+current_date(cs::Interfacer.CoupledSimulation) = cs.t isa ITime ? date(cs.t) : cs.dates.start_date + Dates.second(cs.t)
```

Comment:
```suggestion
current_date(cs::Interfacer.CoupledSimulation) = cs.t isa ITime ? date(cs.t) : cs.start_date + Dates.second(cs.t)
```

`dates` was removed

Reply:
```suggestion
current_date(cs::Interfacer.CoupledSimulation) = cs.t[] isa ITime ? date(cs.t[]) : cs.start_date + Dates.second(cs.t[])
```

And `cs.t[]` is the value

Reply:
Oops! Thank you